TROUBLESHOOTING.md

```
@@ -0,0 +1,410 @@
+# Restart via docker-compose
+make dev
+```
+
+**Agent Action**: `make dev` handles cleanup automatically. Use it instead of manual `docker run`.
+
+---
+
+## Weight Sum Assertion Failure at Startup
+
+### Symptom
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)